types/Context.d.ts

```diff
@@ -0,0 +1,16 @@
+declare type Context = import('@/class/Context').default;
+
+declare type ContextOptions = {
+  oauthClientId: string,
+  oauthServer: string,
+  oauthPopup?: boolean,
+  plugins?: Array<ConnectPlugin>,
+  authUrl?: string,
+  namespace?: string,
+  isIdentityMode?: boolean,
+  widget?: {
+    position?: object,
+  },
+  url: string,
```

Comment:
what is it? from where part `url` is coming?

Reply:
This is legacy. This param now lives in `OauthRequestOptions` type.